package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "issn",
-  "version": "1.0.6",
+  "version": "2.0.0",
```

Comment:
Major version bump because the function signature is changing: the `options` argument is optional but it's still a change.